docs/rtd_environment.yaml

```diff
@@ -0,0 +1,12 @@
+name: rtd311
+channels:
+  - conda-forge
+  - defaults
+dependencies:
+  - astropy
+  - cython
+  - matplotlib
```

Comment:
@mcara Do the drizzle docs depend on Cython or matplotlib? I don't see them imported.

Reply:
I don't know why `cython` or `matplotlib` are listed here. I don't see why they would be.